ReactNativeClient/lib/fs-driver-node.js

```diff
@@ -188,6 +188,10 @@ class FsDriverNode extends FsDriverBase {
 		if (encoding === 'ascii') return buffer.toString('ascii');
 		throw new Error(`Unsupported encoding: ${encoding}`);
 	}
+
+	fileSize(filepath) {
+		return fs.statSync(filepath)['size'];
+	}
```

Comment:
Please use the `stat()` method for this, and you can then get the size in the `size` property. As the code in this file needs to be duplicated for each platform, we try to keep it as lean as possible.